builtins/davix/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ list(APPEND DAVIX_LIBRARIES ${DAVIX_PREFIX}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}da
 list(APPEND DAVIX_LIBRARIES ${DAVIX_PREFIX}/src/DAVIX-build/deps/curl-install/usr/lib/${CMAKE_STATIC_LIBRARY_PREFIX}curl${CMAKE_STATIC_LIBRARY_SUFFIX})
 
 string(REPLACE "-Werror " "" DAVIX_CXX_FLAGS "${CMAKE_CXX_FLAGS} ")
+if (CMAKE_CXX_COMPILER_ID STREQUAL "NVHPC")
```

Comment:
Shouldn't that be upstreamed in the DAVIX repository?

Reply:
Probably. Here is a PR: https://github.com/cern-fts/davix/pull/110

Reply:
A version of Davix with this fix has been release, and the ROOT builtin davix was updated to it.

The first commit of this PR is not needed anymore.